spring-cloud-gcp-pubsub/src/main/java/com/google/cloud/spring/pubsub/core/publisher/PublisherCustomizer.java

```diff
@@ -23,5 +23,5 @@
  * Can be implemented as a lambda accepting a {@link Publisher.Builder} and a `String` topic.
  */
 public interface PublisherCustomizer {
-  public void apply(Publisher.Builder publisherBuilder, String topic);
+  void apply(Publisher.Builder publisherBuilder, String topic);
```

Comment:
Correct me if I'm wrong but methods in `interface` are `public` by default.

Reply:
Ah! you're right. Sorry I didn't realize this was an interface. In fact, this change is actually better since `public` and `abstract` on an interface is already implied.